css/style.css

```diff
@@ -1,9 +1,31 @@
+#menuContainer {
+  z-index: 3;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: baseline;
+  height: 100vh;
+  width: 100vw;
+  background-color: #191765;
+  position: fixed;
+  padding: 70px 10px;
+  transition-duration: 0.5s;
+}
```

Comment:
- [ ] Please ensure the mobile menu follows the Figma design:
- [ ] It would be better to add the close button to your menu so the user can close your Menu.

Your design: 
<img width="212" alt="image" src="https://user-images.githubusercontent.com/77190203/152057973-0b4efe2f-5ff2-43db-a654-eb1e66c2b687.png">
Figma design:
<img width="195" alt="image" src="https://user-images.githubusercontent.com/77190203/152058079-07d80059-f6c0-4c1e-bdb1-e148da0a42fc.png">